### DISCUSSION TITLE
EBook highlight Text

### DISCUSSION THREAD

**Discussion by Hutch79** [advplyr/audiobookshelf#2600] [Ideas]
I come from the Tolino universe, where it is possible to highlight text.
I know this is also possible in the Google Books App.

My suggestion would be the addition to highlight text in a Book and sync the highlights, as well as the possibility to see a list of all highlights.

The benefit of this feature would be an easy way to find where you left reading, across multiple screen sizes and therefore different page sizes.

**Comment by nichwall** [comment]
There is are open feature requests for this. https://github.com/advplyr/audiobookshelf/issues/1558 and https://github.com/advplyr/audiobookshelf-app/issues/1009

**Reply by Hutch79** [reply]
Oh, thanks.
Have only searched the discussions ^^'

I'll close this one and join #1558
